compass/landice/tests/ismip6_run_ais/set_up_experiment.py

```diff
@@ -0,0 +1,215 @@
+import os
+from compass.model import run_model, make_graph_file
+from compass.step import Step
+import shutil, glob, sys
+from jinja2 import Template
+from importlib import resources
+
+
+class SetUpExperiment(Step):
```

Comment:
It seems like this will be a step for setting up and then running an experiment eventually, not just setting it up.

Reply:
The intent was to only use compass for setting up the experiments, but to have users 'manually' submit/run experiments.   The experiments are expensive and complicated enough to only want expert users to be running them.